### DISCUSSION TITLE
MarketplacePurchaseEligibility - Azure Marketplace

### DISCUSSION THREAD

**Discussion by GarethMakgetla** [Security-Onion-Solutions/securityonion#9022] [General] [answered]
I'd like to use the product in a CSP subscription, sent our CSP name and tenant ID via the Contact Us form located on the Security Onion Solutions website but haven't gotten a response.

**Comment by jertel** [comment, marked as answer]
Hello -- We did receive your website request yesterday. Changes in Azure take 1-3 business days to complete and this particular change became active about five minutes ago. You should now have the ability to use the Security Onion image in Azure.

**Comment by GarethMakgetla** [comment]
Hi, Thank you managed to provision VM on Azure
